src/tools/createtool.cpp

```diff
@@ -33,6 +33,8 @@
 DEFINE_string(mds_addr, "127.0.0.1:9000", "mds addr");
 DEFINE_string(file_name, "vdisk_001", "file name");
 DEFINE_uint64(file_size, 1 * 1024 * 1024 * 1024, "file size");
+DEFINE_uint64(stripe_Unit, 32 * 1024, "stripe unit size");
+DEFINE_uint64(stripe_Count, 32, "stripe count");
```

Comment:
it seems this file is no longer in use, you can delete it.

Reply:
fix